schemas/src/elements/inputs/Input.Time.json

```diff
@@ -5,19 +5,19 @@
   "properties": {
     "max": {
       "type": "string",
-      "description": "Hint of maximum value (may be ignored by some clients)."
+      "description": "Hint of maximum value expressed in HH:MM (may be ignored by some clients)."
     },
     "min": {
       "type": "string",
-      "description": "Hint of minimum value (may be ignored by some clients)."
+      "description": "Hint of minimum value expressed in HH:MM (may be ignored by some clients)."
     },
     "placeholder": {
       "type": "string",
       "description": "Description of the input desired. Displayed when no time has been selected."
     },
     "value": {
       "type": "string",
-      "description": "The initial value for this field expressed in ISO-8601 format."
+      "description": "The initial value for this field expressed in ISO-8601 format or in HH:MM."
```

Comment:
>in ISO-8601 [](start = 65, length = 11)

Is there a reason this one is still here as ISO-8601? I though we only supported the simplified times for all of these (UWP only supports HH:MM)

Reply:
thanks for reviewing it; i will update it.

---
In reply to: [364408373](https://github.com/microsoft/AdaptiveCards/pull/3706#discussion_r364408373) [](ancestors = 364408373)